src/plugins/blocksmith/components/twitter-tweet-block-placeholder/index.jsx

```diff
@@ -1,18 +1,24 @@
 import React from 'react';
 import GenericBlockPlaceholder from '@triniti/cms/plugins/blocksmith/components/generic-block-placeholder';
 
-const config = {
-  iconGroup: {
-    icons: {
-      primary: {
-        imgSrc: 'tweet',
-      },
-      secondary: {
-        imgSrc: 'twitter',
+export default (props) => {
+  const { block } = props;
+  const node = block.getData().get('canvasBlock').get('screen_name');
+  const userName = node ? `Tweeted by @${node}` : '';
```

Comment:
@tbone1020 make sure to name things as accurately as possible -

1. the screen_name is not the node, so `const node` is confusing
2. this template literal is not just a userName, so `const userName` is confusing